app/src/test/java/com/findmyrecycling/ProductTests.kt

```diff
@@ -39,7 +39,7 @@ class ProductTests {
     }
 
     @After
-    fun TearDown(){
+    fun tearDown(){
```

Comment:
function names should start with lowercase

Reply:
Yes, in the case of unit tests, they should.